### ISSUE TITLE
Possible clipping

### ISSUE THREAD

**Issue by matoi974** [bozbez/win-capture-audio#6]
When capturing foobar2000 (the music player), I noticed the audio visualizer was turning solid red (clipping) much more often. The attached files may help reproduce this issue, make sure to choose "True Peak" in Settings>Audio>Peak Meter Type.
![image](https://user-images.githubusercontent.com/63289008/129882844-5a78b871-167e-4fad-9575-c43e0ca3c4df.png)

[2021.08.18.zip](https://github.com/bozbez/win-capture-audio/files/7006447/2021.08.18.zip)

**Comment by bozbez** [comment]
It's expected that the audio-capture sources will have higher volume than the desktop audio since we are extracting audio frames _before_ the Windows audio pipeline does its volume control.

Could you reduce the source volume in OBS and report if you can still see/hear clipping?

**Comment by matoi974** [comment]
Yes, after setting the win-capture-audio's volume to -1.3dB, it's still occasionally clipping, whereas capturing the default audio device normally doesn't clip. I verified that there are no DSPs active in foobar2000's settings, and that all software volume sliders are set to 0.0dB.

What's weird is that they're not always clipping in the same part of the track. I hope this recording and log file help demonstrate this.
In this video, audio track №4 contains the audio captured by the plugin, tracks №1 and №3 contain audio captured by obs-studio's inbuilt capture method, and track 2 contains silence.
[2021.08.18.zip](https://github.com/bozbez/win-capture-audio/files/7008140/2021.08.18.zip)

There's no audible clipping for me in either track, but I imagine someone with better hearing might be able to notice it. Or maybe not.
